circle.yml

```diff
@@ -17,12 +17,10 @@ test:
     - make docs
 deployment:
     production:
-        branch: master
+        tag: /v*/
         commands:
-            - make docs
             - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
```

Comment:
Ah do you mean - that only certain tagged commits get push to docs or make sure tagged commits ALSO get put on dev?